assets/js/mosaic-setup.js

```diff
@@ -144,6 +146,26 @@ function customFunctionOnPgnGameLoad() {
     if (IsRotated && getDisplayedMinigame() != displayedGame) {
         flipMiniboard();
     }
+
+    // start clock countdown if enabled
+    if (clockCountdownEnabled) {
+        // get reference time from pgn
+        referenceTime = customPgnHeaderTag("ReferenceTime");
+
+        // remove clockActive if set from previous game
+        document.getElementById("ClockPlace1").classList.remove("clockActive");
+        document.getElementById("ClockPlace2").classList.remove("clockActive");
+
+        if (clockCountdownTimer) {
+            clearInterval(clockCountdownTimer);
+            clockCountdownTimer = undefined;
+        }
+        checkLiveBroadcastStatus();
+        if (LiveBroadcastGamesRunning > 0 && gameResult[currentGame] === '*' && CurrentPly === PlyNumber) {
```

Comment:
Please add newlines before all `if`, `for` etc. & after their scope.
```suggestion
        }
        
        checkLiveBroadcastStatus();
        
        if (LiveBroadcastGamesRunning > 0 && gameResult[currentGame] === '*' && CurrentPly === PlyNumber) {
```